lib/kuiq.rb

```diff
@@ -4,7 +4,7 @@
 
 module Kuiq
   WINDOW_WIDTH = 900.0
-  WINDOW_HEIGHT = 530.0
+  WINDOW_HEIGHT = 560.0
```

Comment:
Does this mean the UI is hard coded in size? Is there a way to make the window and UI dynamically resizable?

Reply:
The width/height are hardcoded for the launch of the application window only. However, the window can definitely be resized by the user, and native controls will know how to resize dynamically, albeit the `area` control needs to know what to do when the window is resized, which I have not programmed yet. I plan to eventually add code that would have the `area` react correctly based on the size of the window. I actually have this task in my personal Kuiq TODO list.

Reply:
Perfect, sounds like an excellent example for people who do want to build complex, resizable UI elements in Glimmer and libUI.